db/migrate/20250522195407_features_plans_index.rb

```diff
@@ -0,0 +1,15 @@
+class FeaturesPlansIndex< ActiveRecord::Migration[7.0]
```

Comment:
Maybe we can wait and see if with Rails 7.2 we can use composite primary key, and we won't need this anymore.

Reply:
We still need it. The database composite index is still there. But it cannot help when searching only by the second component.